package.json

```
@@ -18,7 +18,9 @@
     "test-update-snapshot": "yarn test -- --updateSnapshot",
     "tdd": "jest --watch",
     "lint": "yarn prettier-check && yarn eslint",
+    "lint-fix": "yarn prettier-write && yarn eslint-fix",
     "eslint": "eslint --quiet '+(e|web)/**/*.{ts,tsx,js,jsx,mts}'",
+    "eslint-fix": "eslint --fix --quiet '+(e|web)/**/*.{ts,tsx,js,jsx,mts}'",
     "type-check": "NODE_OPTIONS='--max-old-space-size=4096' tsc",
     "prettier-check": "yarn prettier --check '+(e|web)/**/*.{ts,tsx,js,jsx,mts}'",
     "prettier-write": "yarn prettier --write --loglevel silent '+(e|web)/**/*.{ts,tsx,js,jsx,mts}'",
```

```suggestion
    "prettier-write": "yarn prettier --write --log-level silent '+(e|web)/**/*.{ts,tsx,js,jsx,mts}'",
```

I know you didn't change this, but it's been bugging me that `--loglevel` isn't recognised by the version of prettier we're on, so we might as well make this change here 😀

---

Sure 😄